packages/job-worker/src/playout/moveNextPart.ts

```diff
@@ -23,8 +25,22 @@ export function selectNewPartWithOffsets(
 	if (!refPart || !refPartInstance)
 		throw new Error(`RundownPlaylist "${playlist._id}" has no next and no current part!`)
 
-	const rawSegments = playoutModel.getAllOrderedSegments()
-	const rawParts = playoutModel.getAllOrderedParts()
+	let rawSegments = playoutModel.getAllOrderedSegments()
+	let rawParts = playoutModel.getAllOrderedParts()
+	let allowWrap = false // whether we should wrap to the first part if the curIndex + delta exceeds the total number of parts
+
+	const quickLoopService = new QuickLoopService(_context, playoutModel)
```

Comment:
I'm not sure if this is a desirable solution or if the specific method should be exposed through the PlayoutModel instead.

Reply:
I'm not sure if it should be, but as we have `getSegmentsBetweenQuickLoopMarker` exposed on the `PlayoutModel`, it would be reasonable and consistent to expose this one too.